bm-persona/Libraries/LibraryDetailViewController.swift

```diff
@@ -95,61 +109,71 @@ extension LibraryDetailViewController {
                 addressLabel.text = longAddress
             }
         }
-        card.addSubview(addressLabel)
+        overviewCard.addSubview(addressLabel)
         addressLabel.font = Font.thin(12)
-        addressLabel.numberOfLines = 2
+        addressLabel.numberOfLines = 0
         addressLabel.textColor = UIColor.black
         addressLabel.translatesAutoresizingMaskIntoConstraints = false
-        addressLabel.topAnchor.constraint(equalTo: nameLabel.layoutMarginsGuide.bottomAnchor, constant: 20).isActive = true
-        addressLabel.leftAnchor.constraint(equalTo: addressIcon.layoutMarginsGuide.rightAnchor, constant: 8).isActive = true
-        //addressLabel.widthAnchor.constraint(equalTo: card.layoutMarginsGuide.widthAnchor, multiplier: 0.5).isActive = true
-        addressLabel.rightAnchor.constraint(equalTo: card.layoutMarginsGuide.rightAnchor, constant: -160).isActive = true
+        addressLabel.bottomAnchor.constraint(equalTo: addressIcon.layoutMarginsGuide.bottomAnchor).isActive = true
+        addressLabel.leftAnchor.constraint(equalTo: addressIcon.layoutMarginsGuide.rightAnchor, constant: 15).isActive = true
+        addressLabel.rightAnchor.constraint(equalTo: libPic.layoutMarginsGuide.leftAnchor, constant: -10).isActive = true
+        addressLabel.topAnchor.constraint(equalTo: addressIcon.layoutMarginsGuide.topAnchor).isActive = true
         addressLabel.adjustsFontSizeToFitWidth = true
         
+        let phoneIcon = UIImageView()
+        phoneIcon.image = #imageLiteral(resourceName: "telephone")
+        overviewCard.addSubview(phoneIcon)
+        phoneIcon.translatesAutoresizingMaskIntoConstraints = false
+        phoneIcon.leftAnchor.constraint(equalTo: overviewCard.layoutMarginsGuide.leftAnchor).isActive = true
+        phoneIcon.topAnchor.constraint(equalTo: addressIcon.layoutMarginsGuide.bottomAnchor, constant: 10).isActive = true
+        phoneIcon.widthAnchor.constraint(equalTo: overviewCard.layoutMarginsGuide.widthAnchor, multiplier: 0.06).isActive = true
+        phoneIcon.contentMode = .scaleAspectFit
+        phoneIcon.heightAnchor.constraint(equalToConstant: 20).isActive = true
+        
         
         let phoneLabel = UILabel()
         phoneLabel.text = library!.phoneNumber
-        card.addSubview(phoneLabel)
+        overviewCard.addSubview(phoneLabel)
         phoneLabel.font = Font.thin(12)
```

Comment:
Let's use Font.light instead cause it's super hard to read on the phone. I think all the other thin need to be updated as well. @KevWho @huangshawn1 Is this the same as dining?